brave/brave-core
# fix(wallet): Local Storage Pref Portfolio Filters

## Description 
We now persist `Portfolio Filter` prefs in `Local Storage`


Resolves <https://github.com/brave/brave-browser/issues/27364>

## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Go to the `Portfolio` page
2. Select a `Network` to filter by
3. Select an `Account` to filter by
4. Select a `Asset Filter` type
5. Refresh the page, all filters should persist and not reset.

https://user-images.githubusercontent.com/40611140/216129952-8968521f-b675-4e69-bd61-66a4ff1af912.mov
